Modules/Scripted/SampleData/SampleData.py

```diff
@@ -185,7 +185,7 @@ def setup(self):
     # Add spacer to layout
     self.layout.addStretch(1)
 
-  def logMessage(self,message):
+  def logMessage(self, message, isError=False):
```

Comment:
instead of `isError`, you could have a `logLevel` parameter where value could be `logging.INFO` or `logging.ERROR`. See https://docs.python.org/2/library/logging.html#logging-levels

Reply:
I also suggest to improve `self.log.insertHtml(message)` so that the message is displayed in orange if `logging.WARNING`, and red if above that level.